doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,230 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "aioConnectorMetadataSchemaVersion": "1.0.0",
+  "definitions": {
+    "AdditionalConfiguration": {
+      "type": "object",
+      "description": "The schema for this 'additional configuration' field. Must provide inline schema xor the file that contains the schema",
```

Comment:
It might be worth calling out that this property is settable only for the `managedConfigurationSettings` variant of `runtimeConfiguration` in the `ConnectorTemplate`.